dbms/src/Storages/KVStore/MultiRaft/PrehandleSnapshot.cpp

```diff
@@ -567,7 +563,6 @@ void executeParallelTransform(
         LOG_DEBUG(log, "Try fetch prehandle task split_id={}, region_id={}", extra_id, new_region->id());
         async_tasks.fetchResult(extra_id);
     }
-    if (head_result.error == ReadFromStreamError::Ok)
```

Comment:
Why must we run through the codes below if the "head split" is not OK now?

Reply:
fixed